src/components/drawer/drawer.module.scss

```diff
@@ -0,0 +1,26 @@
+.root {
```

Comment:
Not blocking page scroll is an issue which is also present on the Wix site. Sometimes when I'm trying to scroll the items in the cart the page scrolls instead.

I've never used the <dialog> element, but maybe it could help with that.

Reply:
Disabled scroll using `react-remove-scroll` library